src/workerd/jsg/fast-api.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <workerd/jsg/web-idl.h>
+
+#include <v8-fast-api-calls.h>
+#include <v8-local-handle.h>
+#include <v8-value.h>
+
+#include <kj/common.h>
+
+#include <type_traits>
+#include <utility>
+
+namespace workerd::jsg {
```

Comment:
Nit: I'd like us to get in the habit of including more leading documentation at the top of these headers to provide more context.

Reply:
Yes, please.

A good header contains more comments than code.